### DISCUSSION TITLE
Hoymiles HMT und HM gleichzeitig auslesen

### DISCUSSION THREAD

**Discussion by mika1199** [hoylabs/OpenDTU-OnBattery#705] [General]
Hallo zusammen, 

bin relativ unbedarft mit dem Thema OpenDTU, hab mir nun auf meine Garage eine knapp 8 kWp Anlage gebaut. Durch unterschiedliche Ausrichtungen und Verschattungen hab ich 3x HMT-2250 und 1x HM-400 verbaut.

Mein ESP32-D0WD-V3 hat CMT2300A und nRF24 verbaut, LED SH1106, Kombipacket von Ebay.
Wenn ich das Profil NRF24+CMT2300A auswähle, sollte es ja eigentlich funktionieren, oder? 
Unter Funkmodulinformationen:

nRF24 Status: Konfiguriert
nRF24 Chip-Status: nicht verbunden

CMT2300A funktioniert, kann die HMT sehen

Was mache ich falsch? 
Vielen Dank!
Gruß
Michael

**Comment by mastertape** [comment]
Du solltest dich am den Verkäufer deier DTU kompnenten wenden und nach einer fertigen pin_mapping.json fragen....
Diese kannst du dann unter Einstellungen->Konfigurationsverwaltung einspielen.... Alternativ musst du es selber belegen - es scheint so als anhand deiner Config die anbindung fpr nrf nicht richtig und/oder da vielleicht was falsch verbunden ist (wenss z.b. mit jumperkabeln ist z.B. tx / rx vertauscht?)... Wenns ne steckplatine ist würde ich heher auf ne fehlerhafte pin_mapping.json tippen...

**Reply by mastertape** [reply]
Ansonsten wenn du noch Rückgaberecht hast schick es zurück und kaufe am besten ein von der Community Unterstüttes Board wie dtu-fusion wo alles komplett drauf ist und auch auch die pin mappins im projekt sid erspaart viel huddelei

Ich spreche übrigens mit einer DTU (habe das DTU Fusion Board - 2x HMS-2000 1xHMS1600 1x HM800 an

**Reply by mika1199** [reply]
Danke, hab mir jetzt auch eine dtu-fusion bestellt. 
Viele Grüße
